rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.component_template/10_basic.yml

```diff
@@ -117,8 +117,8 @@
 ---
 "Add data lifecycle":
   - skip:
-      version: " - 8.7.99"
-      reason: "Data Lifecycle template was added after 8.7"
+      version: " - 8.8.99"
```

Comment:
Same as with the code, should we just delete these tests since they'll never run?

Reply:
++